### DISCUSSION TITLE
Example

### DISCUSSION THREAD

**Discussion by jerod33** [alexkn/node-red-contrib-dlna#4] [Q&A]
Hi,
 is it possible to add an example of using this node? Thanks in advance.
 Jarda

**Comment by alexkn** [comment]
Please describe exactly what you tried. Maybe then I can help you.

**Reply by jerod33** [reply]
Hi. I had problems with my LG-TV. I thought that an example would guide me to solve it. Unfortunately the problem persists, my tv refuses to connect.
